config/routes.rb

```diff
@@ -25,7 +25,9 @@
     resources :reviews
   end
 
-  resources :orders, only: %i[index show new]
+  resources :carts, only: %i[index show] do
+    get 'show_order', to: 'carts#show_order', as: 'show_order', on: :member
```

Comment:
please, show order should be in `OrdersController#show`

Reply:
Added OrdersController and route to the ```OrdersController#show```